infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -110,6 +110,28 @@ public LogReplicationMetadataManager(CorfuRuntime rt, long topologyConfigId, Str
         setupTopologyConfigId(topologyConfigId);
     }
 
+    public void initializeReplicationStatusTable() {
```

Comment:
any reason this initialization is not done from the constructor?

Reply:
Yes, this is actually invoked in the constructor of CorfuReplicationManager to ensure that this happens only once and only on the source side.